.eslintrc

```diff
@@ -33,7 +33,7 @@
     "comma-spacing": [2, { "before": false, "after": true }],
     "comma-style": [2, "last"],
     "complexity": 0,
-    "consistent-return": 2,
+    // "consistent-return": 2,
```

Comment:
Why are we commenting out all these rules? If they are a bad idea, we should remove them. If they are a good idea and only sometimes we need to break them, then we can add comments to the infringing code to disable the offending eslint rule there.

Reply:
I will remove those I think are not good rules